crates/block-producer/src/main.rs

```diff
@@ -127,6 +128,20 @@ fn run_cli() -> Result<()> {
 
 /// Godwoken entry
 fn main() {
-    env_logger::init_from_env(env_logger::Env::default().default_filter_or("info"));
+    init_log();
     run_cli().expect("run cli");
 }
+
+fn init_log() {
+    let logger = env_logger::builder()
+        .parse_env(env_logger::Env::default().default_filter_or("info"))
+        .build();
+    let level = logger.filter();
+    let logger = sentry_log::SentryLogger::with_dest(logger).filter(|md| match md.level() {
+        log::Level::Error => LogFilter::Event,
```

Comment:
Should we also logging warning to sentry?

Reply:
Let me try it.

Reply:
Just added. We don't have a lot of warning logs. Looks fine.